modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/milker.dm

```diff
@@ -0,0 +1,319 @@
+					self_message = span_purple("You wrench [being_milked]'s breast up to your mouth and start to bite down!"),
+					blind_message = span_purple("You hear a chomping noise."),
+					vision_distance = 1)
+				else
+					milker.visible_message(
+					message = span_purple("[milker] clasps a hand into one of [being_milked]'s breasts, then plunges [milker.p_their()] mouth around [being_milked.p_their()] nipple!"),
+					self_message = span_purple("You grab onto one of [being_milked]'s breasts and ram that nipple right into your mouth!"),
+					blind_message = span_purple("You hear a plapping noise."),
+					vision_distance = 1)
+
+	if(!do_mob(milker, being_milked, 1 SECONDS))
+		milker.visible_message(
+		message = span_purple("[milker] was interrupted!"),
+		self_message = span_purple("You were interrupted!"),
+		blind_message = span_purple("You hear someone's lips slip."),
+		vision_distance = 1)
+		return FALSE
+	if(!breast_check(being_milked, milker, breasts)) // Their breasts may have changed state (gutted, clothes, drained...)
+		return FALSE
+
+	milker.adjustArousal(arousal_amount)
+
+	switch(handle_mode)
+		if(HANDLE_LIGHT) // Gentle paws...
+			if(milker.combat_mode) // Nibble...
+				milker.visible_message(
+				message = span_purple("[milker] nibbles on [self_suckle ? "[milker.p_their()] own" : "[being_milked]'s"] nipple, teasing out a stream of milk!"),
+				self_message = span_purple("You nibble into [self_suckle ? "your own" : "[being_milked]'s"] nipple and feel a stream of milk spray into your mouth!"),
+				blind_message = span_purple("You hear a nibble, and a squirt."),
+				vision_distance = 1)
+				being_milked.adjustPain(pain_amount)
+				playsound(milker.loc,'sound/weapons/bite.ogg', rand(10,50), TRUE)
+			else
+				milker.visible_message(
+				message = span_purple("[milker] suckles out a gentle stream of [self_suckle ? "[milker.p_their()] own" : "[being_milked]'s"] milk!"),
+				self_message = span_purple("You suckle down a gentle stream of [self_suckle ? "your own" : "[being_milked]'s"] milk!"),
+				blind_message = span_purple("You hear a slurp."),
+				vision_distance = 1)
+				being_milked.adjustArousal(arousal_amount)
+				being_milked.adjustPleasure(pleasure_amount)
+				playsound(milker.loc,'sound/items/drink.ogg', rand(10,50), TRUE)
+		if(HANDLE_STRONG) // SQUISH
+			if(milker.combat_mode) // CHOMP
+				milker.visible_message(
+				message = span_purple("[milker] bites down into [self_suckle ? "[milker.p_their()] own" : "[being_milked]'s"] nipple, slurping down a mouthful of milk!"),
+				self_message = span_purple("You chomp down into [self_suckle ? "your own" : "[being_milked]'s"] breast and feast upon the milk within!"),
+				blind_message = span_purple("You hear a hungry nibble."),
+				vision_distance = 1)
+				being_milked.adjustPain(pain_amount)
+				playsound(milker.loc,'sound/weapons/bite.ogg', rand(10,50), TRUE)
```

Comment:
```suggestion
				playsound(milker.loc, 'sound/weapons/bite.ogg', rand(10, 50), TRUE)
```
You can use `,([^ \\])` as your search and `, $1` as your replace if you want to quick check this PR.

Reply:
Spaces after commas, sure!